app/controllers/admin/schedules_controller.rb

```diff
@@ -3,7 +3,7 @@
 
 module Admin
   class SchedulesController < ResourceController
-    before_filter :adapt_params, only: [:create, :update]
+    before_filter :adapt_params, only: [:update]
```

Comment:
I wonder if we should just copy the update method from resource controller as well, this line would be removed and it would become:

```ruby
      def update
        adapt_params
        if @object.update_attributes(permitted_resource_params)
          flash[:success] = flash_message_for(@object, :successfully_updated)
          respond_with(@object) do |format|
            format.js   { render layout: false }
          end
        else
          respond_with(@object)
        end
      end
```

and if we bring the destroy method like this:
```ruby
      def destroy
        if @object.destroy
          flash[:success] = flash_message_for(@object, :successfully_removed)
          respond_with(@object) do |format|
            format.js   { render partial: "spree/admin/shared/destroy" }
          end
        else
          respond_with(@object) do |format|
            format.html { redirect_to collection_url }
          end
        end
      end
```

and then :tada: 
class SchedulesController < Spree::Admin::BaseController

It's for another PR I think.

Reply:
Yes, the more `before_filter` and callbacks I remove is becoming evident that the inheritance doesn't provide much. Indeed, for an upcoming PR.